WFP-VAM/prism-app
# Sync opacity between layer and legend controls

### Description

This fixes #1080 .

Adds a central store for opacity, so it's value and way to update it is consistent between components.
@wadhwamatic one change is that the opaicty is persisted when we switch layers off/on. This is on puporse but can be changed if we prefer.

## How to test the feature:

- enable any layer and make sure the value on the legend is the same with the left panel
- generally play around with opacity

## Checklist - did you ...



Test your changes with


## Screenshot/video of feature:
